GameServer/Logic/Config.cs

```diff
@@ -33,7 +33,7 @@ private void LoadConfig(string path)
             foreach (var player in playerConfigurations)
             {
                 var playerConfig = new PlayerConfig(player);
-                Players.Add(string.Format("player{0}", Players.Count + 1), playerConfig);
+                Players.Add($"player{Players.Count + 1}", playerConfig);
```

Comment:
I do think this is more convenient than string formatting, but to make the code guidelines clear here, I think we should exclude all inline logic from strings. So a better way to do this would be:
```CS
var playerNum = Players.Count + 1;
Players.Add($"player{playerNum}", playerConfig);
```